### PR TITLE
fix: changing light to dark gray 

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5951

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/styles/libs/_helpers.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 }
 
 .text.muted {
-  color: #888 !important;
+  color: #404040 !important;
```
Comment (iamareebjamal) [inline app/styles/libs/_helpers.scss:16]: 444 would be better

**Comment by iamareebjamal** [comment]
image
image

File: app/styles/partials/event-card.scss [hunk 2]
```diff
@@ -26,6 +26,11 @@
     white-space: nowrap;
     width: 90%;
   }
+
+  .link.item {
+    color: #555 !important;
```
Comment (iamareebjamal) [inline app/styles/partials/event-card.scss:31]: That changes every link item's color

Comment (maze-runnar) [reply app/styles/partials/event-card.scss:31]: it's changing only session-card, and event-card link color as it is used inside `.extra.content`.

**Comment by mariobehling** [comment]
@maze-runnar Please see comment here https://github.com/fossasia/open-event-frontend/issues/5951#issuecomment-750012125. A number of areas are still light gray. Please help to make reviews faster and easier and always post screenshots of changes. Thanks.

**Pull request merged**